datawig/mxnet_input_symbols.py

```diff
@@ -102,15 +103,15 @@ class LSTMFeaturizer(Featurizer):
     def __init__(self,
                  field_name: str,
                  seq_len: int = 500,
-                 vocab_size: int = 40,
```

Comment:
I'd rather not introduce backwards incompatible breaking changes, if not absolutely required. These names were changed a couple times back and forth and it's really a matter of taste what people think is most intuitive. In fact one of the initial choices for this parameter has been ``max_tokens``

Reply:
Backwards compatibility is a good point. It's not a matter of taste, however, but of consistency.  max_tokens and vocab_size are the same thing. BowEncoder takes max_tokens as input, BowFeaturizer takes vocab_size. In fact these two parameter should always be identical, right?

Reply:
I agree it's not consistent - but all these names were cr'd previously and I remember that correctly this inconsistency was introduced to make things more consistent amongst the Featurizer classes. 

I'm just trying to say that if we break the API we should really have good reasons for this.

Reply:
Alternative I would write a workaround and internally assign the parameter names to comply with the inconsistent naming scheme. Happy to do that. 

I can not see any reason for _consistency among feature classes_ (?)